usagov/vote-gov-drupal
# VOTE-2078: Resolve config import issue for Site Builder role

## Jira ticket

[VOTE-2078](https://cm-jira.usa.gov/browse/VOTE-2078)

## Description

Update configurations to resolve import errors for Site Builder role.

## Deployment and testing

### Post-deploy steps

None

### QA/Testing instructions

1. Locally run `lando retune`
2. Edit your web/sites/settings.local.php file and set the following
 $config['config_split.config_split.production']['status'] = TRUE;
3. Run `lando drush cim` and confirm that the import runs successfully.
4. Edit your web/sites/settings.local.php file and set the following
 $config['config_split.config_split.non_production']['status'] = TRUE;
5. Run `lando drush cim` and confirm that the import runs successfully.
6. Edit your web/sites/settings.local.php file and set the following
 $config['config_split.config_split.local']['status'] = TRUE;
7. Run `lando drush cim` and confirm that the import runs successfully.

## Checklist for the Developer




## Checklist for the Peer Reviewers
